src/game_battler.cpp

```diff
@@ -326,7 +326,7 @@ bool Game_Battler::UseSkill(int skill_id, const Game_Battler* source) {
 		}
 		effect *= mul;
 
-		effect += (effect * Utils::GetRandomNumber(-skill->variance * 10, skill->variance * 10) / 100);
+		if (Player::IsRPG2k() || effect > 0) effect = Game_Battle::VarianceAdjustEffect(effect, skill->variance);
```

Comment:
Checking `effect > 0` is correct, but again I don't see this being skipped for 2k

Reply:
This "sometimes effect of 1 despite calculated effect of zero"-case even occurs on healing skills outside of battles. But here I have to change the check to RPG Maker 2000 versions older than 1.50 too.

Reply:
Added version check for RPG Maker 2000 version below 1.50 now.